src/index.html

```diff
@@ -11,7 +11,174 @@
+              maxlength="100"
+              minlength="1"
+              id="number"
+            >
+          </label>
+        </div>
+
+        <div class="fieldset">
+          <label for="dateOfBirth">
+            Full date of birth:
+            <input type="date">
+          </label>
+        </div>
+
+        <div class="fieldset">
+          <label for="term">
+            I accept the term of the agreement
+            <input type="checkbox">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <div>
+          <label for="emails">
+            E-mail:
+            <input
+              type="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="fieldset">
+          <label for="password">
+            Password:
+            <input
+              type="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+        <div>
+          <label for="loveCatsyes">
+            Do you love cats?
+            <input
+              type="radio"
+              name="Yes"
+            >
+              Yes
+          </label>
+
+          <label for="loveCatsno">
+            <input
+              type="radio"
+              name="No"
+            >
+              No
+          </label>
+        </div>
+
+        <div class="fieldset">
+          <label for="fovoriteColor">
+            What is your favorite color?
+            <input type="color">
+          </label>
+        </div>
+
+        <div class="fieldset">
+          <label for="timeGoBed">
+            What time do you go to bed?
+            <input
+              type="time"
+              min="01:00"
+              max="00:00"
+              placeholder="--:--:--"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="fieldset">
+          What are your favorite brand of cars?
+          <select
+            class="selector"
+            name="Cars"
+            multiple="Audi,BMW,Lada"
+          >
+            <option value="BMW" selected>BMW</option>
+            <option value="Audi" selected>Audi</option>
+            <option value="Lada" selected>Lada</option>
+          </select>
+        </div>
+
+        <div class="fieldset">
+          <label for="rateWork">
+            How do you rate our work?
+            <input
+              type="range"
+              value="0"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Additional info:</legend>
+        <div>
+          Comments:
+          <textarea name="comments"autocomplete="off"></textarea>
+        </div>
+
+        <div class="fieldset">
+          Would you recommend us?
+          <select
```

Comment:
p3 [Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

Reply:
not fixed